src/main/java/view/FrameHistoryParser.java

```diff
@@ -0,0 +1,52 @@
+package view;
+
+import domain.Pins;
+import spark.utils.CollectionUtils;
+
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.Objects;
+import java.util.stream.Collectors;
+
+public class FrameHistoryParser {
+
+    private static Map<Integer, String> replaces = new HashMap<>();
+
+    static {
+        replaces.put(10, "X");
+        replaces.put(0, "-");
+    }
+
+    private FrameHistoryParser() {
+    }
+
+    public static String parse(List<Integer> frameHistories) {
+        if (CollectionUtils.isEmpty(frameHistories)) {
+            return "";
+        }
```

Comment:
아래와 같이 바꾸는게 가독성상 좋다고 생각이 들어요!
```suggestion
        if (CollectionUtils.isEmpty(frameHistories)) {
            return StringUtils.EMPTY;
        }
```